nmkapp/views.py

```diff
@@ -177,6 +177,26 @@ def request_display_name(request):
 
     return render(request, 'request_display_name.html', {'form': form, 'no_menu': True})
 
+@login_required
+@transaction.atomic
+def create_password(request):
+    logger.info('User is on change password page')
+
+    user = request.user
+
+    if user and user.has_usable_password():
+        return redirect(reverse(password_change))
+
+    if request.method == 'POST':
+        form = SetPasswordForm(user, request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect(reverse(home))
```

Comment:
da ga vratimo na profil mozda? ne znam, samo bacam ideju?

Reply:
Pa on ga automatski izloguje kad postavi sifru. Ne znam, mislim da nema potrebe, dovoljno je da mu damo neko obavestenje da je uspesno postavio sifru

Reply:
automatski ga izloguje? nisam video taj deo. Kako si to izveo?:o "home" je samo za ulogovane korisnike, isto kao i "profil"?

Reply:
rekao bih da to django sam uradi ako je ulogovanom korisniku promjena sifra. Ja nisam to stavljao